straxen/plugins/event_processing.py

```diff
@@ -766,6 +755,18 @@ class EnergyEstimates(strax.Plugin):
     ] + strax.time_fields
     save_when = strax.SaveWhen.TARGET
 
+    # config options
+    # g1 and g2 values default to 1T values
+    g1 = straxen.URLConfig(
+        default=0.1426,
+        help="S1 gain in PE / photons produced")
+    g2 = straxen.URLConfig(
+        default=11.55 / (1 - 0.63),
+        help="S2 gain in PE / electrons produced")
+    lxe_w = straxen.URLConfig(
+        default=13.7e-3,
+        help="LXe work function in quanta/keV")
```

Comment:
@ershockley don't we want to make a start with #784 and use the nT defaults here?

Also, the 1T defaults are not correct for all 1T runs, we should make them run dependent. @jmosbacher what would be the cleanest way of doing this with the URL configs?

Reply:
@JoranAngevaare maybe add a protocol called e.g. defaults1t that accepts the run_id and returns the defaults? then you can set the configs for 1t to all be `defaults1t://name?run_id=plugin.run_id`

Reply:
Sounds good, I think such hard coding within straxen would be fine for 1T. Things aren't going to change there anyway.

In theory it could be bad if we update those `defaults1t` with different values, as the lineage wouldn't change. Then again, how big is our focus on 1T really in straxen.

Reply:
Yeah if you wanted to change the defaults in the future you would have to use a new protocol name since once you set a protocol name you commiting to it always giving back the same value for the same inputs. If you think there is a reasonable chance of this happening then just add version parameter to the url  set to 0/1 and for now ignore it in the protocol but if you want to change the defaults in the future you use a new version for the new values. The upgrades to CMT in #816 should make editing the protocol to use version in the future (if needed) very easy

Reply:
To be honest I don't think we should bother with the 1T stuff, it's a bit of a waste of time in my view. I left the 1T values as defaults because (I'm ashamed to say) I don't know how to use bodega to get the nT values 😕 

@JosephJHowlett or someone can you show me how to get the nT values here in a secret way?

Reply:
I can see this point of view too, but it's almost trivial to make our changes carefully and make sure that if someday someone wants to use 1T data, this person doesn't have to re-invent the wheel.

for your second question
```python
nT_numbers = straxen.get_resource("XENONnT_numbers.json", fmt="json")
g1 = nT_numbers["g1"]["v2"]["value"]
g2 = nT_numbers["g2"]["v2"]["value"]
```

Reply:
Ok thanks, I will implement the nT defaults (though we should probably talk about putting these numbers into CMT also, the functionality w/ versions etc is almost identical it seems to me). 

If you guys want to implement the 1T stuff please go ahead.

Reply:
>Ok thanks, I will implement the nT defaults (though we should probably talk about putting these numbers into CMT also, the functionality w/ versions etc is almost identical it seems to me).

From what I understood, this was indeed the idea https://github.com/XENONnT/private_nt_aux_files/pull/61#issuecomment-832988529

Reply:
> If you guys want to implement the 1T stuff please go ahead.

If our ACs don't care about these things, I'm not going to swim upstream because that would be fighting a lost battle 😂 😉

Reply:
Yeah I remember that discussion, I think maybe we should have it again. g1 and g2 in particular I feel like should be in CMT since they might have some time dependence.

Reply:
Are those values actually used? If that is the case we should probably incorporate to the corrections workflow as kind of final result ..

Reply:
@l-althueser yes I agree